spec/spec_helper.rb

```diff
@@ -25,3 +25,21 @@ module SpecHelper
 RSpec.configure do |config|
   config.filter_run_when_matching :focus
 end
+
+def set_circle_env(define_ci)
```

Comment:
If my suggestion above to use `CI` was implemented, this could become.

```suggestion
def set_ci_env(define_ci)
```